pkg/inttest/http.go

```diff
@@ -185,6 +185,7 @@ func (hc *HTTPClient) SignIn(t *testing.T, email, password string) (*http.Cookie
 	require.Equal(t, http.StatusCreated, res.StatusCode, errMsg+": HTTP status mismatch")
 
 	body, err := io.ReadAll(res.Body)
+	require.NoError(t, err)
```

Comment:
please add a message like we do in the other requires in the helper `errMsg+"..."`
try to make your tests fail, look at the error and see how you can improve it